feconf.py

```diff
@@ -490,7 +490,8 @@ def get_empty_ratings():
         'ImageClickInput',
         'ItemSelectionInput',
         'MultipleChoiceInput',
-        'TextInput'
+        'TextInput',
+        'DragAndDropSortInput'
```

Comment:
Hasn't this been enabled already?

Reply:
Yes, it has been enabled already. Previous commits came in this PR.

Reply:
So maybe merge this with latest develop branch. So this commits would be removed.